msteams-platform/concepts/build-and-test/app-studio-overview.md

```diff
@@ -25,15 +25,15 @@ App Studio is a Teams app, which can be found in the Teams store. Follow this li
 
 In the store, search for App Studio.
 
-:::image type="content" source="../../assets/images/get-started/StoreTeamsAppStudio.png" alt-text="Store entry for app studio":::
+:::image type="content" source="../../assets/images/get-started/StoreTeamsAppStudio.png" alt-text="The screenshot is an example that shows the store entry for app studio.":::
```

Comment:
What's with the casing?
Check globally across all alt-text updates. CC @v-habibaumme

Reply:
changed the casing